graphql/operations.graphql

```diff
@@ -46,118 +46,79 @@ query PatientsList($options: PatientOptions, $where: PatientWhere) {
   }
 }
 
-query SamplesList(
-  $where: SampleWhere
-  $options: SampleOptions
-  $sampleMetadataOptions: SampleMetadataOptions
-  $bamCompletesOptions: BamCompleteOptions
-  $mafCompletesOptions: MafCompleteOptions
-  $qcCompletesOptions: QcCompleteOptions
-) {
-  samplesConnection(where: $where) {
+query DashboardSamples($searchVals: [String], $sampleContext: SampleContext) {
```

Comment:
FastLoadSamples ? open to other names as well

Reply:
We're no longer using the original query anywhere in the dashboard, as of this PR. Would you still like to rename this new query?